src/bp/lang-server/service/index.ts

```diff
@@ -29,13 +38,13 @@ export default class LanguageService {
     this._discoverModels()
 
     console.log(
-      `Loading languages "${this.listModels()
```

Comment:
Is there a logger service in the lang server? We should create one instead of console logging directly. So we can add more context to the log

Reply:
The language server code is in the botpress codebase, however it starts independently, so there is no logger actually for that (inversify isn't used for such a small task)